Core/MIPS/x86/Asm.cpp

```diff
@@ -162,7 +163,6 @@ void Jit::GenerateFixedCode(JitOptions &jo) {
 			// IMPORTANT - We jump on negative, not carry!!!
 			FixupBranch bail = J_CC(CC_S, true);
 
-			SetJumpTarget(skipToRealDispatch);
 			SetJumpTarget(skipToRealDispatch2);
```

Comment:
Well, yeah. I'll change it the next time there's something in the dispatcher to fix :)